api/src/main/resources/workbench.yaml

```diff
@@ -775,12 +775,16 @@ definitions:
     required:
       - code
```

Comment:
Let's update this now that `code` is a nonexistent property

Reply:
yep.. i will update all.